### PR TITLE
docs: unset

### PR DESCRIPTION
**Pull request by soyeric128**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

added doc for UNSET.

Closes #issue

### CODE REVIEW

**Comment by TCeason** [comment]
Link impl https://github.com/datafuselabs/databend/pull/8870

File: docs/doc/14-sql-commands/80-setting-cmds/unset.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Examples
+
+This example assigns new values to some settings, changes their levels to GLOBAL, then resets them to their defaults:
+
+```sql
+---Show default values
+SELECT name, value, default, level from system.settings where name in ('sql_dialect', 'timezone', 'wait_for_async_insert_timeout');
+
+---
```
Comment (TCeason) [inline docs/doc/14-sql-commands/80-setting-cmds/unset.md:23]: Please delete line with——

File: docs/doc/14-sql-commands/80-setting-cmds/unset.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: UNSET
+---
+
+Sets one or more settings back to their default values. The settings will also be reset to the initial SESSION level if they were set to GLOBAL level. See [Examples](#examples) for how to reset a GLOBAL setting with UNSET.
```
Comment (TCeason) [inline docs/doc/14-sql-commands/80-setting-cmds/unset.md:5]: And will reset the global level to session level.

File: docs/doc/14-sql-commands/80-setting-cmds/unset.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+```sql
+UNSET <setting_name> | ( <setting_name> [, <setting_name> ...])
+```
+
+## Examples
+
+This example assigns new values to some settings, changes their levels to GLOBAL, then resets them to their defaults:
+
+```sql
```
Comment (TCeason) [inline docs/doc/14-sql-commands/80-setting-cmds/unset.md:19]:
```suggestion
```sql
-- Show default values
mysql> SELECT name, value, default, level from system.settings where name in ('sql_dialect', 'timezone', 'wait_for_async_insert_timeout');
+-------------------------------+------------+------------+---------+
| name                          | value      | default    | level   |
+-------------------------------+------------+------------+---------+
| sql_dialect                   | PostgreSQL | PostgreSQL | SESSION |
| timezone                      | UTC        | UTC        | SESSION |
| wait_for_async_insert_timeout | 100        | 100        | SESSION |
+-------------------------------+------------+------------+---------+

-- Set new values
mysql> SET GLOBAL sql_dialect='MySQL';

mysql> SET GLOBAL timezone='Asia/Shanghai';

mysql> SET GLOBAL wait_for_async_insert_timeout=20000;


mysql> SELECT name, value, default, level from system.settings where name in ('sql_dialect', 'timezone', 'wait_for_async_insert_timeout');
+-------------------------------+---------------+------------+--------+
| name                          | value         | default    | level  |
+-------------------------------+---------------+------------+--------+
| sql_dialect                   | MySQL         | PostgreSQL | GLOBAL |
| timezone                      | Asia/Shanghai | UTC        | GLOBAL |
| wait_for_async_insert_timeout | 20000         | 100        | GLOBAL |
+-------------------------------+---------------+------------+--------+

-- Reset to default value and level
mysql> UNSET (timezone, wait_for_async_insert_timeout);

mysql> UNSET sql_dialect;

mysql> SELECT name, value, default, level from system.settings where name in ('sql_dialect', 'timezone', 'wait_for_async_insert_timeout');
+-------------------------------+------------+------------+---------+
| name                          | value      | default    | level   |
+-------------------------------+------------+------------+---------+
| sql_dialect                   | PostgreSQL | PostgreSQL | SESSION |
| timezone                      | UTC        | UTC        | SESSION |
| wait_for_async_insert_timeout | 100        | 100        | SESSION |
+-------------------------------+------------+------------+---------+


```

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**